dashboard/static/css/layout.css

```diff
@@ -7,9 +7,31 @@ body {
     margin: 0 0 100px;
 }
 
+main {
+    margin-top: 20px;
+    margin-bottom: 20px;
+}
+
+a {
+    color: #00549E;
+}
```

Comment:
So far, we've been using *layout.css* to style major page elements like the header and the footer. To keep with that approach, we could move this into a separate *main.css* file.

MDN does something similar, putting layout styles in *structure.styl* and element styles in *elements.styl*. But we don't really need to get quite that fine-grained since we're not using a pre-processor yet.